spec/components/app_triage_form_component_spec.rb

```diff
@@ -21,4 +21,116 @@
 
     it { should have_css(".app-fieldset__legend--reset") }
   end
+
+  describe "hint text and triage options for consented delivery method(s)" do
+    before do
+      allow(programme).to receive(:has_multiple_delivery_methods?).and_return(
```

Comment:
I'm wondering if we can avoid stubbing methods and use FactoryBot instead. The majority of the other component tests using FactoryBot to build state. Not a showstopper, but worth considering.